comparison.py

```diff
@@ -0,0 +1,334 @@
+"""
+
+Author: Damian Pietroń
+Created: 2024-03-18
+Last modified: 2024-03-18
+
+Description: This script takes a path
+to a folder with the data and calculates
+the HRV parameters for the ABP and ECG signals.
+The calculated parameters are then merged into
+a single dataframe and saved as a csv file.
+The script also creates Bland-Altman plots for
+the calculated parameters.
+
+"""
+import neurokit2 as nk
+from pyhrv import frequency_domain as fd
+import matplotlib.pyplot as plt
+from pathlib import Path
+import pandas as pd
+import os
+import numpy as np
+import scipy.signal as sp_sig
+import scipy.fftpack as sp_fft  
+from tqdm import tqdm
+from datetime import datetime
+# The following lists contain the HRV parameters that will be calculated for the ABP and ECG signals.
+CONTENT = [
+    'HRV_SDNN',
+    'HRV_RMSSD',
+    'HRV_MeanNN',
+    'HRV_pNN50',
+    'HRV_pNN20',
+    'HRV_ApEn',
+    'HRV_SampEn',
+    'HRV_FuzzyEn',
+    'HRV_MSEn',
+    'HRV_ShanEn'
+]
+
+FREQ_CONTENT = [
+    'HRV_LF',
+    'HRV_HF',
+    'HRV_TP'
+]
+
+COLUMN_SETS_ECG = [
+    ['ekg[]', 'co2[co2]', 'etco2[mmHg]'], 
+    ['ecg', 'co2[mmHg]', 'etco2[mmHg]'],
+    ["ekg[ekg]", "co2[co2]", "etco2[etco2]"],
+    ['ekg[]', "co2[mm_Hg]", "etco2[mm_Hg]"]
+    ]
+
+COLUMN_SETS_ABP = [
+    ['abp_finger[abp_finger]', "co2[mmHg]", "etco2[mmHg]"], 
+    ['abp_cnap[mmHg]', "co2[mmHg]", "etco2[mmHg]"], 
+    ["abp_finger[abp_finger]", "co2[co2]", "etco2[etco2]"],
+    ["abp_finger[mm_Hg]", "co2[mm_Hg]", "etco2[mm_Hg]"]
+    ]
+
+def handle_path(path):

```

Comment:
Add [type hints](https://docs.python.org/3/library/typing.html) for all of the functions, as follows:

```python
from typing import List, Tuple

def handle_path(path: str) -> Tuple[List[Path], Path]:
```